content/docs/buildpack-author-guide/create-buildpack/building-blocks-cnb.md

```diff
@@ -2,20 +2,16 @@
 +++
 title="Building blocks of a Cloud Native Buildpack"
 weight=402
-creatordisplayname = "Scott Sisil"
-creatoremail = "ssisil@pivotal.io"
-lastmodifierdisplayname = "Scott Sisil"
-lastmodifieremail = "ssisil@pivotal.io"
 +++
 
-Now we will set up the buildpack scaffolding. You will need to make these files in your `ruby-cnb` directory
+Now we will set up the buildpack scaffolding. You will need to make these files in your `ruby-buildpack` directory
 
 ```bash
-cd ~/workspace/ruby-cnb
+cd /tmp/ruby-buildpack
```

Comment:
I find `/tmp` a kind of weird place to do this. Maybe we should do something like this:

```
mkdir ruby-buildpack
cd ruby-buildpack
```

That way it doesn't matter where they base it.
```

Reply:
Done. I also updated reference to files, when updating the contents, to be relative to the root working dir so that `cd` wasn't a thing of complexity.